src/DynamoCore/Visualization/DefaultRenderPackageFactory.cs

```diff
@@ -545,7 +545,7 @@ public void AddTextureMapForMeshVerticesRange(int startIndex, int endIndex, byte
         /// This flag is used by the UpdateRenderPackageAsyncTask implementation to flag
         /// any third party usage of deprecated color methods in IRenderPackage API
         /// </summary>
-        [Obsolete("Do not use! This will be removed in Dynamo 3.0")]
+        [Obsolete("Do not use! This will be removed in a future version of Dynamo")]
         public bool AllowLegacyColorOperations { get; set; } = true;
```

Comment:
internal?

Reply:
Its is not possible in this case, is part of an interface.

Reply:
I think it's okay to remove and break the interface, but I think really only @saintentropy would be able to do this in a timely fashion, so I guess it's okay to keep it.

Reply:
Yes, I am keeping this.